### PR TITLE
te: adding changes to provide merged results on the dashboard

### PR DESCRIPTION
**Pull request by puneetjaiswal**
adding changes to provide merged results on the dashboard

### CODE REVIEW

File: thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java [hunk 1]
```diff
@@ -107,37 +113,31 @@ public AnomalyResource(ThirdEyeDashboardConfiguration dashboardConfiguration, An
       throw new IllegalArgumentException("dataset is a required query param");
     }
 
-    List<AnomalyResult> anomalyResults = new ArrayList<>();
+    DateTime endTime = DateTime.now();
+    if (StringUtils.isNotEmpty(endTimeIso)) {
+      endTime = ISODateTimeFormat.dateTimeParser().parseDateTime(endTimeIso);
+    }
+    DateTime startTime = endTime.minusDays(21);
```
Comment (kishoreg) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java]: why r  we hard coding 21 days

Comment (puneetjaiswal) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java]: It was for testing purpose, will remove it.

**Comment by kishoreg** [comment]
lgtm apart from the hard coded days

**Comment by MarthaWB** [comment]
LGTM

**Comment by puneetjaiswal** [comment]
reverted default startTime to currentTime - 7 days

**Pull request merged**